src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -50,7 +51,7 @@ export class UBAClientWithRefresh extends BaseUBAClient {
       )
     );
   }
-  public get isUpdated(): boolean {
+  public get areClientsUpdated(): boolean {
```

Comment:
I was getting the error
```
Cannot set property `isUpdated` which has only a getter
```

I think this was because we overrode the `isUpdated` getter of the parent `BaseUBAClient` class and therefore the `isUpdated` setter somehow got inaccessible. I fixed this by using a different method name

Reply:
I think @james-a-morris has a fix for this can you rebase master?

Reply:
Unfortunately the fix didn't work... I ended up with an error `Maximum Call stack`.

Reply:
ok changed it bakc